nextflow.config

```diff
@@ -123,6 +128,7 @@ profiles {
   test_diff_hash { includeConfig 'conf/test_diff_hash.config' }
   test_diff_hash_abundance { includeConfig 'conf/test_diff_hash_abundance.config' }
   test_diff_hash_sourmash { includeConfig 'conf/test_diff_hash_sourmash.config' }
+  test_diff_hash_is_aligned { includeConfig 'conf/test_diff_hash_is_aligned.config' }
```

Comment:
seems like some params are being added but not updated in usage/readme/help and the config printing section